src/Presentation/Presenters/PageNotFoundPresenter.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation\Presenters;
+
+use WMDE\Fundraising\Frontend\Presentation\TwigTemplate;
+
+/**
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+class PageNotFoundPresenter {
```

Comment:
This class seems a bit silly. How about just inlining this code until there is more than just a single method call ?